app/views/teacher_interface/registration_number/edit_ghana.html.erb

```diff
@@ -0,0 +1,37 @@
+<% content_for :page_title, title_with_error_prefix(t("application_form.tasks.items.registration_number.ghana"), error: @form.errors.any?) %>
+<% content_for :back_link_url, back_history_path(default: teacher_interface_application_form_path) %>
+
+<%= form_with model: @form, url: %i[registration_number teacher_interface application_form] do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <%= f.govuk_text_field :registration_number, label: { size: "l", tag: "h1" } %>
+
+  <% if @application_form.region.status_check_online? %>
```

Comment:
I can't see any issues with this but could we go through status_check_online and status_information just for my own understanding. I'm not sure what these are and how they relate to the workflow of the website.